_CoqProject.in

```diff
@@ -1,3 +1,4 @@
+META.coq-fiat-parsers
```

Comment:
This is not backwards compatible.  Instead the Makefile should be updated to insert this line only on versions of Coq new enough to support it

Reply:
can you point me to the piece of code you want to me patch? Do you already test the version of Coq?

Reply:
You should add an `echo` to
https://github.com/mit-plv/fiat/blob/cb85ee76ee3410f04333633f6644663ebb525ac8/Makefile#L376-L377
in addition to adding it here, and then modify the code starting at https://github.com/mit-plv/fiat/blob/cb85ee76ee3410f04333633f6644663ebb525ac8/Makefile#L54 to extend `EXTRA_PIPE_SED_FOR_COQPROJECT ` at
https://github.com/mit-plv/fiat/blob/cb85ee76ee3410f04333633f6644663ebb525ac8/Makefile#L50
with something like `+= | sed 's/^META.*$$//g'` in old versions of Coq.

Reply:
done.